### DISCUSSION TITLE
45.Pytorch training loops - learning rate and the gradient chart

### DISCUSSION THREAD

**Discussion by lucifer3618** [mrdbourke/pytorch-deep-learning#597] [Q&A]
img  = [https://static.javatpoint.com/tutorial/machine-learning/images/gradient-descent-in-machine-learning2.png](url)

In above img it shows that in each epoch, optimizer takes different size of step towards the minimum loss. How it happens while learning rate is set to a some constant value.
